targets/python/source/source/PlayFabHTTP.py

```diff
@@ -0,0 +1,36 @@
+import json
+import PlayFabSettings
+import PlayFabErrors
+import requests
+
+def DoPost(urlPath, request, authType, authKey, extraHeaders):
```

Comment:
Probably worth documenting that this is a blocking call, and potentially the return type as well.